seqr/views/apis/dataset_api_tests.py

```diff
@@ -344,3 +343,50 @@ def test_add_alignment_sample(self, mock_local_file_exists, mock_subprocess):
             {sample_guid}
         )
         mock_subprocess.assert_called_with('gsutil ls gs://readviz/NA19679.bam', stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=True)
+
+"""
+# Tests for AnVIL access disabled
+class LocalDatasetAPITest(AuthenticationTestCase, DatasetAPITest):
+    fixtures = ['users', '1kg_project']
+"""
+
+
+def assertions(self, call_count):
```

Comment:
can you rename this something a bit ore descriptive, like `assert_has_anvil_calls`